In [20]:
from pprint import pprint


CP_HOST = 'https://localhost:8080'

In [23]:
import requests
session = requests.Session()
session.verify = False
response = session.post(url='https://127.0.0.1:8080/iserver/auth/status')
print(response.status_code)
# pprint(response.json())

404


I logged in to https://localhost:8080, redirected to https://localhost:8080/sso/Dispatcher, which shows me the message 'Client login succeeds'.  After this part, I tried three ways to get meaningful response from https://localhost:8080/iserver/auth/status.  In particular, I wanted to see a JSON as described https://interactivebrokers.github.io/cpwebapi/authentication.

First attempt was with cURL:
```bash
❯ curl -X POST --insecure -v https://localhost:8080/iserver/auth/status
*   Trying 127.0.0.1:8080...
* Connected to localhost (127.0.0.1) port 8080 (#0)
* ALPN: offers h2,http/1.1
* (304) (OUT), TLS handshake, Client hello (1):
* (304) (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
* TLSv1.2 (IN), TLS handshake, Server finished (14):
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
* TLSv1.2 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.2 (OUT), TLS handshake, Finished (20):
* TLSv1.2 (IN), TLS change cipher, Change cipher spec (1):
* TLSv1.2 (IN), TLS handshake, Finished (20):
* SSL connection using TLSv1.2 / ECDHE-RSA-AES256-GCM-SHA384
* ALPN: server did not agree on a protocol. Uses default.
* Server certificate:
*  subject: C=US; ST=CT; L=Greenwich; O=Interactive Brokers; OU=Client Portal; CN=Client Portal Web API
*  start date: Feb 10 16:26:02 2019 GMT
*  expire date: May 11 16:26:02 2019 GMT
*  issuer: C=US; ST=CT; L=Greenwich; O=Interactive Brokers; OU=Client Portal; CN=Client Portal Web API
*  SSL certificate verify result: self signed certificate (18), continuing anyway.
* using HTTP/1.x
> POST /iserver/auth/status HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.88.1
> Accept: */*
>
< HTTP/1.1 302 Found
< Location: https://api.ibkr.com/iserver/auth/status
< Transfer-Encoding: chunked
<
* Connection #0 to host localhost left intact
```

I also could not even resolve the API with Python:
```python 
import requests
session = requests.Session()
session.verify = False
response = session.post(url='https://127.0.0.1:8080/iserver/auth/status')
print(response.status_code)
``` 
which gives me a 404.

I also tried navigating my browser to https://localhost:8080/iserver/auth/status, which immediately redirects to https://api.ibkr.com/iserver/auth/status and prints the following message:
```
Not Found
The requested URL was not found on this server.

Additionally, a 404 Not Found error was encountered while trying to use an ErrorDocument to handle the request.
```